packages/grafana-ui/src/components/Link/TextLink.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { Router } from 'react-router-dom';
+
+import { GrafanaConfig, locationUtil } from '@grafana/data';
+import { locationService } from '@grafana/runtime';
+
+import { TextLink } from './TextLink';
+
+describe('TextLink', () => {
+  const link = 'http://www.grafana.com/grafana/after-sub-url';
+  beforeEach(() => {
+    const { location } = window;
+    // @ts-ignore
+    delete window.location;
```

Comment:
There's an [easier way](https://github.com/grafana/grafana/blob/a30f8645d1a0d0c5c7525ef69080a0dc8681d5da/contribute/style-guides/testing.md#mocking-the-window-object-and-its-methods) to mock `window`.

Reply:
Ooh nice, thanks!